js/header_dialog.js

```diff
@@ -964,6 +1030,22 @@ function HeaderDialog(dialog, onSave) {
                         }
                 });
 
+                // Scan the pid_bandwidth table
+                $(".pid_bandwidth input").each(function() {
+                        if($(this).val()!=null) {
+                                if($(this).attr('decPl')!=null) {
+                                        var matches=$(this).attr('name').match(/(.+)\[(\d+)\]/);
+                                        if(matches!=null) {
+                                                if(newSysConfig[matches[1]]==null) newSysConfig[matches[1]] = [null, null, null];
+                                                var newArray = newSysConfig[matches[1]];
+                                                newArray[matches[2]] = (parseFloat($(this).val()) * Math.pow(10, $(this).attr('decPl')));
+                                        } else (parseFloat($(this).val()) * Math.pow(10, $(this).attr('decPl')));
```

Comment:
what is this line doing? the () instead of {}?

Reply:
This function is not required and does not get used. Removing.